shotover-proxy/src/transforms/cassandra/sink_cluster/mod.rs

```diff
@@ -753,6 +811,45 @@ enum RewriteTableTy {
     Peers,
 }
 
+fn get_prepared_result_message(
+    message: &mut Message,
+) -> Result<Option<(CBytesShort, PreparedMetadata)>> {
```

Comment:
This never returns an `Err` so we should remove the Result from the return value.